packages/mui-material/src/Accordion/Accordion.test.js

```diff
@@ -210,4 +212,45 @@ describe('<Accordion />', () => {
       'MUI: A component is changing the uncontrolled expanded state of Accordion to be controlled.',
     );
   });
+
+  it('should merge the TransitionsProps with default theme TransitionProps props', () => {
```

Comment:
This test is passing on `master` HEAD, so it's probably not testing what's needed.